docker-alpine/Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM alpine:latest
+
+#Otherwise mosquitto fails
+VOLUME ["/sys/fs/cgroup"]
+
+#We need curl to get root certificates
+RUN apk update
+RUN apk upgrade
+RUN apk add mosquitto
+RUN apk add curl
```

Comment:
You comment that we need curl to get certs but it doesn't appear anywhere in the files that it is being used.

Reply:
`curl` package provides a lot of certificates to `/etc/ssl/certs`. It's for convenience. Without installing it, you need to give a correct certificate for c8y or az.